test/trivial/deitz/expressions/all_precedence_levels_in_a_single_expression.chpl

```diff
@@ -2,8 +2,8 @@ class D { var x = 1; }
 class C: D { var y = 2; }
 operator **(d:borrowed D,i:int) do return d.x..i;
 
-write("C(): ");
-writeln(borrowed C():string);
+write("C: ");
+writeln(borrowed C:string);
```

Comment:
These changes probably aren't actually necessary, right?  (not that I object to them).  I'm curious... how did you come upon them in developing this PR?   (e.g., were there other changes required in this file for a time that were then reverted?)

Reply:
Without the change, I get a warning for `borrowed C()`:

```
all_precedence_levels_in_a_single_expression.chpl:6: warning: unnecessary type construction call
all_precedence_levels_in_a_single_expression.chpl:2: note: 'C' has no generic fields
all_precedence_levels_in_a_single_expression.chpl:6: note: remove the ()
C(): borrowed C
```

Note that, currently, declarations such as `record R { var x = 1; }` produce a concrete type (and not a generic-with-defaults one). It is the same in the test but we are dealing with a class.

I did have other changes in the file but Lydia pointed out that this change is simpler.

Reply:
Ah, makes sense, thanks.  I was focused almost exclusively on the first of these two lines and couldn't figure out why the string within the writeln() had to change.  I think when typing this, I was incorrectly mis-reading the second line as also being within a string literal.  (and now that I see it's not, I agree that it makes sense to have the string literal match).